connectors/sync_job_runner.py

```diff
@@ -171,6 +164,49 @@ async def execute(self):
             if self.data_provider is not None:
                 await self.data_provider.close()
 
+    async def _execute_access_control_sync_job(self, job_type, bulk_options):
+        if requires_platinum_license(self.sync_job, self.connector, self.source_klass):
+            (
+                is_platinum_license_enabled,
+                license_enabled,
+            ) = await self.elastic_server.has_active_license_enabled(License.PLATINUM)
+
+            if not is_platinum_license_enabled:
+                raise InsufficientESLicenseError(
+                    required_license=License.PLATINUM, actual_license=license_enabled
+                )
+
+        await self.elastic_server.async_bulk(
+            self.sync_job.index_name,
+            self.generator(),
+            self.sync_job.pipeline,
```

Comment:
`pipeline` doesn't apply to access control sync? As `async_bulk` and `Sink` are used for both content and access control sync, we should probably make the param `pipeline` optional:

https://github.com/elastic/connectors-python/blob/cfb12ee86ececbeb9ff479b1d23ace64f8208ae4/connectors/es/sink.py#L729-L733

https://github.com/elastic/connectors-python/blob/cfb12ee86ececbeb9ff479b1d23ace64f8208ae4/connectors/es/sink.py#L82-L87